Makefile

```diff
@@ -78,4 +78,4 @@ publish:
 
 
 dubbed-video-csv: pex
-	./makecontentpacks ./contentpacks/update_dubbed_video_spreadsheet.py
+	./makecontentpacks ./contentpacks/update_dubbed_video_spreadsheet.py  --version=0.16.x
```

Comment:
Removing the hard-coded to 0.16 and use `KALITE_BUILD_VERSION` to get the build version of the contentpack-maker.